--- ./boost_install/include/boost/container_hash/hash.hpp	2025-08-07 12:03:13
+++ ./boost_install/include/boost/container_hash/hash.hpp.new	2025-08-07 12:03:34
@@ -129,7 +129,7 @@
         };
 #else
         template <typename T>
-        struct hash_base : std::unary_function<T, std::size_t> {};
+        struct hash_base  {};
 #endif
 
         struct enable_hash_value { typedef std::size_t type; };
